tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -312,13 +311,14 @@
     ],
 )
 
+
 child_two = RowSchema(
     document=TANF_T3DataSubmissionDocument(),
-    quiet_preparser_errors=True,
+    quiet_preparser_errors=validators.is_quiet_preparser_errors(101),
```

Comment:
Having this be two different types scares me a bit. Is there something we could do to avoid that?

Reply:
The only way I can think of is to use the function in all other cases

Reply:
Ya, that was my thought also. Im curious what others will think on this.